pkg/common/namespace.go

```diff
@@ -7,16 +7,19 @@ type NamespaceMapping int
 const namespaceFormat = "%s-%s"
 
 const (
-	ProjectDomain NamespaceMapping = iota
-	Domain        NamespaceMapping = iota
+	NmProjectDomain NamespaceMapping = iota
+	NmDomain        NamespaceMapping = iota
+	NmProject       NamespaceMapping = iota
```

Comment:
`Project` is already declared in the package, so I just prefixed all of these with `Nm`. Let me know if there is a preferred practice around this.